htmresearch/regions/LanguageSensor.py

```diff
@@ -104,6 +106,13 @@ def getSpec(cls):
       },
       "inputs":{},
       "parameters":{
+        "cacheRoot":{
```

Comment:
B/c it wasn't clear the cacheDir param for the encoder was being set properly, so this seemed to fix it, but this is a bad fix. The region shouldn't care about the cache; the cache should be set by the encoder and handled by the Cio client.